### ISSUE TITLE
Auth routes are not triggering eAuth

### ISSUE THREAD

**Issue by mtlaney** [18F/fs-open-forest-platform#591]
## Notes
When navigating to /admin/applications or /admin/christmas-trees eauth is not being triggered to load.
Generating a 401 from fs-intake-api-trees.app.cloud.gov/auth/user and stopping.
Only on staging and prod, not locally

## Acceptance Criteria
- [x] navigating to /admin/applications when not logged in should trigger eAuth login
- [x] navigating to /admin/christmas-trees should trigger eAuth login

## Definition of Done
- [x] Pull requests meet technical defintion of done
- [x] Circle CI Tests Passed
- [x] Work has been reviewed by someone other than the author
- [x] PO Approved

**Comment by ASprinkle** [comment]
@davemcorwin I am checking through the Zenhub catching up. I posted this comment on #540 but perhaps it's more applicable here: "As a side note: upon attempting to log in to the Open Forest staging site to test this, I was prompted to log in through login.gov.
Once logged in, and viewing the Christmas tree staging landing page, I did not see the admin functions.
![image](https://user-images.githubusercontent.com/33435621/55891241-1136ab80-5b69-11e9-99ef-23eb9c46a98f.png) "
